cmake/templates/conf.py.in

```diff
@@ -595,16 +595,26 @@ htmlhelp_basename = 'HPXdoc'
 
 # -- Options for LaTeX output ---------------------------------------------
 
-latex_elements = {}
+latex_elements = {
+    'preamble': r'''
+    \usepackage{enumitem}
+    \setlistdepth{2000}
+    \nonstopmode
+    ''',
+    'maxlistdepth': '100',
+    'releasename': ' ', # NOTE: Needs the space. Empty string means default.
+}
 
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title,
 #  author, documentclass [howto, manual, or own class]).
 latex_documents = [
     (master_doc, 'HPX.tex', u'HPX Documentation',
-     u'The Ste\\textbar{}\\textbar{}ar Group', 'manual'),
+     u'The STE\\textbar{}\\textbar{}AR Group', 'manual'),
 ]
```

Comment:
FWIW, we've used `{STE\raisebox{.05em}{\hspace{.1em}$|$\hspace{0.1em}$|$\hspace{.02em}}AR}` before to typeset `STE||AR`

Reply:
That's awfully specific, but I can change that ;) Thanks!

Reply:
There is nothing more important than names and logos! ;-)